src/test/ui/closures/closure-return-type-mismatch.stderr

```diff
@@ -1,6 +1,8 @@
 error[E0308]: mismatched types
   --> $DIR/closure-return-type-mismatch.rs:7:9
    |
+LL |         let a = true;
```

Comment:
Hm, I'm not convinced that pointing out the type of a literal at the place a variable is initialized is very useful compared to other expressions or places further away from where the variable is declared...

Reply:
I have two options: do not show any span labels if the only thing it's doing is pointing at the declaration, or ignore literal expressions. Either way, it might not be interesting as a way to understand the inference steps, *but* it works as a "found `bool` because of this init value".